crawler/src/main/java/ir/jimbo/crawler/App.java

```diff
@@ -21,11 +21,11 @@
 
     private static final Logger LOGGER = LogManager.getLogger(App.class);
     private static ArrayBlockingQueue<String> queue;
-    private static RedisConfiguration redisConfiguration;
-    private static KafkaConfiguration kafkaConfiguration;
-    private static AppConfiguration appConfiguration;
-    private static LinkConsumer[] consumers;
-    private static PageParserThread[] producers;
+    static RedisConfiguration redisConfiguration;
```

Comment:
add access level

Reply:
access set to private and setter and getter added